### DISCUSSION TITLE
Why does earTrumpet have years of WhatsApp, Teams and Discord .exe's stored?

### DISCUSSION THREAD

**Discussion by finia2NA** [File-New-Project/EarTrumpet#985] [Q&A] [answered]
Hi,
while freeing up space on my system I saw that for some reason, eartrumpet has years worth of WhatsApp, Discord, Teams, Opera and more .exe's stored in my local App data. Is this normal? And if so, just out of curiosity... why?

![image](https://user-images.githubusercontent.com/33845166/147567719-831e075f-6dac-4ea1-8bf1-f1bf5d34d188.png)

Thanks!

**Comment by riverar** [comment, marked as answer]
These are hard links and not real copies of those files and the result of a bug in Windows. (Right-click > Properties > look at Size on disk).

See also https://github.com/File-New-Project/EarTrumpet/issues/257
